README.md

```diff
@@ -4,55 +4,37 @@ Translate Writing3D xml projects into Unity projects. Each scene file is automat
 
 ## Install
 
-```console
+```sh
```

Comment:
Who are these instructions for? If only developers, then install via `poetry` is okay, but If we think that anyone could install the package, then ideally they could instead do 
`pipx install git+https://github.com/brown-ccv/w3d-translator`

and  futhermore w3d_translator.py could be configured as a console script so one just needs to do something along lines

```
w3d_translate indir outdir [OPTIONS]
```

This doesn't need to be addressed now, but I often find helpful to separate the normal user instructions from the development instructions

Reply:
They're for developers, which is really just us?

I'm curious who may end up running the script? Is that something we expect John Caley to do at some point or are we going to translate all of the projects?